sci-biology/ugene/ugene-1.15.1.ebuild

```diff
@@ -22,14 +22,11 @@ RDEPEND="${DEPEND}"
 LANGS="cs en ru zh"
 
 src_configure() {
-	local CONFIG_OPTS
-	if use amd64; then
-		CONFIG_OPTS+=( CONFIG+="x64" )
-	elif use ppc; then
-		CONFIG_OPTS+=( CONFIG+="ppc" )
-	fi
+	CONFIG_OPTS=(
+		usex amd64 x64 ""
+		usev ppc
+		usex cpu_flags_x86_sse2 use_sse2 ""
```

Comment:
sorry for wasting your time, I just decided to more closely look at the ebuild, and I noticed how it depends on qt, so I guess I'll just leave it to @stharward and you guys, since I certainly will not be using it.